packages/app/src/components/Root/Root.tsx

```diff
@@ -250,51 +250,61 @@ export const Root = ({ children }: PropsWithChildren<{}>) => {
   return (
     <>
       <ApplicationHeaders position="above-sidebar" />
-      <SidebarPage>
-        <ApplicationHeaders position="above-main-content" />
-        <Sidebar>
-          <SidebarLogo />
-          <SidebarGroup label="Search" icon={<SearchIcon />} to="/search">
-            <SidebarSearchModal />
-          </SidebarGroup>
-          <SidebarDivider />
-          <SidebarGroup label="Menu" icon={<MuiMenuIcon />}>
-            {/* Global nav, not org-specific */}
-            {renderMenuItems(true, false)}
-            {/* End global nav */}
+      <Box sx={{ height: '100vh', overflow: 'auto', scrollbarWidth: 'none' }}>
```

Comment:
Hmmm... :thinking:

Not sure about this change. I'm just worried that it could introduce issues.

Reply:
I had shown her this scroll bar issue for which she might have added this change

![double-scroll](https://github.com/user-attachments/assets/fb2eed95-1877-419e-8d8c-75395873952c)
after that, I no longer see the double scroll bar appearing

Reply:
This is the fix for two scrollbar. It avoids to place the appbar inside of the `html`. It's harder to set html styles to hide the scrollbar. I verified in this PR it doesn't have problems for without header or with header.

Reply:
Hi, for me your fix didn't worked. I felt wired when scrolling because there was still 2 scrollbars, also if one wasn't visible. It's hard to explain:

https://github.com/user-attachments/assets/eaccbb88-ade3-44e4-be3c-ccc474272668

I added this commit https://github.com/redhat-developer/rhdh/pull/2263/commits/b314f0887cfdea75ce662167b87d0d8e3df8d812 but don't like to merge it without at least one of you have tried it as well.

I think it also fixed https://issues.redhat.com/browse/RHIDP-4637, which you can see if you have a techdocs and view that via docs > doc or via catalog > entity > docs tab

Wdyt?